src/components/AttentionBox/AttentionBox.module.scss

```diff
@@ -8,6 +8,11 @@
   width: fit-content;
   max-width: 100%;
   height: fit-content;
+
+  // Overriding on primary icon color should be with bigger specificity than the original color
+  & .closeIconWrapper .closeIcon {
+    color: var(--primary-text-color);
```

Comment:
I know we usually prefer flat decorations but here we need to override the white text color of on primary button (we don't have alternative for lighter background colors)